packages/suite/src/components/suite/modals/ApplicationLog.tsx

```diff
@@ -21,26 +20,24 @@ const LogWrapper = styled.pre`
     word-break: break-all;
 `;
 
-type Props = {
-    onCancel: () => void;
-};
+type ApplicationLogProps = { onCancel: () => void };
 
-export const Log = ({ onCancel }: Props) => {
+export const ApplicationLog = ({ onCancel }: ApplicationLogProps) => {
     const htmlElement = createRef<HTMLPreElement>();
+    const [hideSensitiveInfo, setHideSensitiveInfo] = useState<boolean>(false);
```

Comment:
`<boolean>` not needed ;]

Reply:
b439a1b4aa7011d5c4cc5bf0e87d09562f056a5f